### PR TITLE
Merged branch 'master' of ezsystems/ezplatform-automated-translation into 4.5

### PR DESCRIPTION
**Pull request by mateuszdebinski**
cross merge: https://github.com/ezsystems/ezplatform-automated-translation/pull/29

### CODE REVIEW

File: tests/lib/Encoder/Field/PageBuilderFieldEncoderTest.php [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+use Ibexa\Contracts\FieldTypePage\FieldType\Page\Block\Definition\BlockDefinition;
+use Ibexa\Contracts\FieldTypePage\FieldType\Page\Block\Definition\BlockDefinitionFactory;
+use PHPUnit\Framework\TestCase;
+
+final class PageBuilderFieldEncoderTest extends TestCase
+{
+    public const ATTRIBUTE_VALUE = 'ibexa';
+
+    /** @var \Ibexa\AutomatedTranslation\Encoder\BlockAttribute\BlockAttributeEncoderManager */
+    private $blockAttributeEncoderManagerMock;
```
Comment (konradoboza) [inline tests/lib/Encoder/Field/PageBuilderFieldEncoderTest.php:29]:
```suggestion
    private BlockAttributeEncoderManager $blockAttributeEncoderManagerMock;
```

File: tests/lib/Encoder/Field/PageBuilderFieldEncoderTest.php [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+    /** @var \Ibexa\Contracts\FieldTypePage\FieldType\Page\Block\Definition\BlockDefinitionFactory */
+    private $blockDefinitionFactoryMock;
```
Comment (konradoboza) [inline tests/lib/Encoder/Field/PageBuilderFieldEncoderTest.php:32]:
```suggestion
    private BlockDefinitionFactory $blockDefinitionFactoryMock;
```

File: tests/lib/Encoder/Field/PageBuilderFieldEncoderTest.php [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+use Ibexa\Contracts\FieldTypePage\FieldType\Page\Block\Definition\BlockDefinition;
+use Ibexa\Contracts\FieldTypePage\FieldType\Page\Block\Definition\BlockDefinitionFactory;
+use PHPUnit\Framework\TestCase;
+
+final class PageBuilderFieldEncoderTest extends TestCase
+{
+    public const ATTRIBUTE_VALUE = 'ibexa';
+
+    private BlockAttributeEncoderManager $blockAttributeEncoderManagerMock;
+    private BlockDefinitionFactory $blockDefinitionFactoryMock;
```
Comment (konradoboza) [inline tests/lib/Encoder/Field/PageBuilderFieldEncoderTest.php:29]:
```suggestion
    
    private BlockDefinitionFactory $blockDefinitionFactoryMock;
```

**Review by konradoboza — APPROVED** [review]

**Review by mnocon — APPROVED** [review]

**Comment by konradoboza** [comment]
@mateuszdebinski please merge up changes to `main`.

**Pull request merged**